### PR TITLE
Historical bounty upload

### LINKED ISSUE #1327: Reconcile missing historic bounties
It was raised that pre-GitHub database vulnerabilities/bounties are missing, meaning that Timeline activities are not being tracked on some profiles.

Thanks to @ferretwithaberet for raising this issue!

e.g. 039-js-valib and 053-js-vizion are old IDs that were not translated over to the new DB.

We need to:

1. Reconcile all previous vulnerabilities from the old database that are missing
2. Create vulnerability.json and README.md files for each
3. Associate the correct Discloser and Fixer IDs
4. Add these events to the Timeline database

We should look to have this done by the end of the week - cheers!

### PR DESCRIPTION
**Pull request by benharvie**
These bounties were missing from the switch over to GitHub. A `vulnerability.json` and `README.md` has been created for each and the relative `Fixer` ID has been appended.

Missing bounties rectified:
```
1-npm-harp
1-rubygems-parsel
1-npm-pm2
1-rubygems-erubis
1-pip-ipycache
1-pip-python-libnmap
1-npm-express-laravel-passport
1-npm-fileview
1-npm-autolinker.js
1-rubygems-dragonfly
1-npm-sceditor
1-npm-node-df
1-npm-node-js-php-unserialize
1-npm-node-gitlab-hook
1-npm-larvitbase-www
1-npm-mavonEditor
1-npm-wxchangba
1-npm-clearance
1-npm-ansi2html
1-npm-editor.md
1-npm-node-pdf-image
1-npm-curling
1-npm-listening-processes
1-npm-simple-crypto-js
1-other-django-crm
1-other-typecho
1-npm-aws-cli-js
1-npm-monitor
1-npm-logkitty
1-npm-ios-simulator
1-npm-benchmark.js
1-npm-node-unrar
1-npm-node-dig-dns
1-npm-node-virtualbox
1-npm-node-gry
1-npm-node-is-git-remote
1-npm-node-git-lib
1-npm-node-git-wrapper
1-other-extanalysis
1-npm-apikit
1-npm-class-transformer
1-maven-azkaban
1-npm-paypal-adaptive-sdk-nodejs
1-npm-wysiwyg-editor
1-npm-svg2png
```

Closes #1327

Cheers 👻

### CODE REVIEW

**Comment by JamieSlome** [comment]
@benharvie - can you take a look at the conflict, plus - we need to identify all of the records in our database that need updating, i.e. switching out old IDs to reflect the new ID structure for the constructed `vulnerability.json` and `README.md` in your pull request.

Half way there! 🍰

**Comment by JamieSlome** [comment]
@benharvie - let's get a little checklist together:

- [ ] Validate editor.md
- [ ] Update out-of-date IDs in both database tables
- [ ] Remove erroneous GitHub Search results returned by the old query in Timeline

**Comment by benharvie** [comment]
This closes #1327, as for updating the database tables, I will move this to a separate issue in relation to `Contributions` and `Timeline` correlation. These bounties will be required on the repo to more easily debug and remedy this separate issue.

Cheers 👻

**Comment by JamieSlome** [comment]
Regarding, the `editor.md`, it looks like two separate disclosures?

**Comment by JamieSlome** [comment]
@benharvie - awesome, thanks buddy!

File: bounties/npm/editor.md/2/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "Stars": "10596"
+  },
+  "Permalinks": [""],
+  "References": [
+    {
+      "Description": "",
+      "URL": ""
+    }
+  ],
+  "PrNumber": "962"
```
Comment (JamieSlome) [inline bounties/npm/editor.md/2/vulnerability.json:53]: Change to `443` - thanks! 🍰

**Pull request merged**